fedora/jellyfin.spec

```diff
@@ -71,47 +72,62 @@ dotnet publish --configuration Release --self-contained --runtime %{dotnet_runti
 
 
 %install
-%{__mkdir} -p %{buildroot}%{_libdir}/%{name} %{buildroot}%{_bindir}
-%{__cp} -r Jellyfin.Server/bin/Release/net6.0/%{dotnet_runtime}/publish/* %{buildroot}%{_libdir}/%{name}
-
-%{__install} -D -m 0644 %{SOURCE15} %{buildroot}%{_sysconfdir}/systemd/system/jellyfin.service.d/override.conf
-%{__install} -D -m 0644 %{SOURCE17} %{buildroot}%{_unitdir}/jellyfin.service.d/jellyfin-server-lowports.conf
-%{__install} -D -m 0644 Jellyfin.Server/Resources/Configuration/logging.json %{buildroot}%{_sysconfdir}/jellyfin/logging.json
-%{__mkdir} -p %{buildroot}%{_bindir}
-tee %{buildroot}%{_bindir}/jellyfin << EOF
-#!/bin/sh
-exec %{_libdir}/jellyfin/jellyfin \${@}
-EOF
+# Jellyfin files
+%{__mkdir} -p %{buildroot}%{_libdir}/jellyfin %{buildroot}%{_bindir}
```

Comment:
Why are you reverting `%{name}` to `jellyfin` here (and elsewhere)?

Reply:
I had used `%{name}` only for those two lines and it was inconsistent with the rest of the file.

I was thinking about replacing `jellyfin` everywhere but I decided against it since I didn't know which of them were hardcoded in the .NET code and which were variable.

Essentially this just brings it in line with the rest of the file. The [current version](https://github.com/jellyfin/jellyfin/blob/a2abae3014578e8d9c7f1b54140fe3dcf1503ad2/fedora/jellyfin.spec) uses `%{name}` only for the relation to other pacakges (server and server-lowports).